packages/cmrjs/tests/cmr-utils/test-reconcilecmrmetadata.js

```diff
@@ -350,7 +350,7 @@ test('publishUMMGJSON2CMR calls ingestUMMGranule with ummgMetadata via valid CMR
     granuleId: 'fakeGranuleID',
   };
   const publishUMMGJSON2CMR = cmrUtils.__get__('publishUMMGJSON2CMR');
-  const ingestFake = sinon.fake.resolves({ result: { 'concept-id': 'fakeID' } });
+  const ingestFake = sinon.fake.resolves({ 'concept-id': 'fakeID', 'revision-id': 123 });
```

Comment:
Is it necessary to alter this mock to return a `revision-id`?

Reply:
It's not and I don't have a good reason for adding it except I thought the next person might find it helpful to know that it could return a revision id.